app/main/views/metrics.py

```diff
@@ -188,7 +188,7 @@ def get_application_metrics_csv():
     si = io.StringIO()
 
     writer = csv.writer(si, delimiter=',', quotechar='"')
-    writer.writerow(list(metrics.keys()))
+    writer.writerow(metrics.keys())
```

Comment:
Does this need to be wrapped in `list()`?  `metrics.keys()` will return a [view object](https://docs.python.org/3.3/library/stdtypes.html#dict-views) which may not work if passed to `writerow()`.

Reply:
Resolved